src/overview/cc-header-app.js

```diff
@@ -372,21 +374,10 @@ export class CcHeaderApp extends LitElement {
 
         .commit {
           align-items: flex-start;
+          color: var(--color-text-default);
           display: flex;
         }
 
-        .commit[data-type="git"] {
-          color: #5D5D5D;
-        }
-
-        .commit[data-type="running"] {
-          color: #2faa60;
-        }
-
-        .commit[data-type="starting"] {
-          color: #2b96fd;
-        }
```

Comment:
we removed these because icon colors are still legacy and text is not.
We thought using color for text was not very useful here anyway since icons already convey the status.
Difference before / after in screenshots below:
![image](https://user-images.githubusercontent.com/100240294/160849294-9389b418-874f-4454-9a77-44a7f48b7ef7.png)
![image](https://user-images.githubusercontent.com/100240294/160849313-ceadd1d2-af01-4b82-a89b-1d725553d1be.png)

Reply:
:+1: